ReviewBot.py

```diff
@@ -825,6 +900,13 @@ def alarm_called(nr, frame):
 
             break
 
+        if not msg:
+            msg = 'done'
```

Comment:
Why don't we just expose service level monitoring data as those tools already exist and are more than sufficient as this effectively gives us "last ran X minutes ago". Should just setup `monitor.opensuse.org` or w/e and install any monitoring service.

wheel 2.0

Reply:
yes and no. there already is monitoring somewhere used by the admins. That one does all kinds of things like checking disk space, pending updates etc and is not a convenient way to just show what is needed for release management.

however, you are right that the way I implemented here may not be the best one. Another alternative that I have in mind is to use sd_notify (https://pypi.org/project/sdnotify/, https://www.freedesktop.org/software/systemd/man/sd_notify.html) in ReviewBot to notify systemd instead.  Then another rather simple program could monitor selected systemd services and send that over to rabbitmq. That way the framework in ReviewBot would be useful for other monitoring tools too.

Reply:
Sure the one used by IT is configured to do more, but a separate instance could be configured to do less. Or heck move all heroes machine monitoring there as I would imagine that would make sense since contributors likely cannot see the monitoring data.

Collecting status information is one of those solved problems and the inevitable notify me when status is out of bounds and such has already been figured out elsewhere.

For instance, including the logs would allow anyone to determine if a completely run worked properly. This is just a lot of engineering to answer the question "when did service X last run" considering how generic that question is.